client/webserver/site/src/js/doc.js

```diff
@@ -213,6 +213,17 @@ export default class Doc {
     return fullPrecisionFormatter(prec).format(v)
   }
 
+  /*
+  * formatConventionalUnits formats the value in atomic units into a
+  * string representation in conventional units, without localizing
+  * the resulting string. If the value happens to be an integer,
+  * no decimals are displayed. Trailing zeros may be truncated.
+  */
+  static formatConventionalUnits (vAtomic, unitInfo) {
+    const [v] = convertToConventional(vAtomic, unitInfo)
+    return v
+  }
+
```

Comment:
Just need to divide by the conventional conversion factor. No need for a function.

```Javascript
const v = vAtomic / unitInfo.conventional.conversionFactor
```